src/systems/hero1_simple_movement.rs

```diff
@@ -22,7 +22,7 @@ pub fn hero1_simple_movement_system(
         let flee = flee.unwrap();
         let leader_team = leader_team.unwrap();
         let caught = caught.unwrap();
-        if leader.0 == 1 {
+        if leader.0 == 1 || leader.0 == 2 {
```

Comment:
I don't think we need this check, as only leaders will have the `Hero1SimpleMovement` component.
Also, we can make a way of choosing which leaders will have this component, if we need to.

Reply:
That makes sense; I have removed it.